yb_migrate/src/migration/common.go

```diff
@@ -97,6 +97,9 @@ func UpdateTableRowCount(source *utils.Source, exportDir string, tablesProgressM
 		}
 
 		queryFrom := tablesProgressMetadata[key].FullTableName
```

Comment:
Can't we set the `FullTableName` to the `schema.table` format--so that users of the field won't have to bother about it again?

Reply:
@amit-yb  FullTableName already the format of `schema.table`
Ididn't get your question?

Reply:
Then why do we need to set the `queryFrom` to the `schema.table` format on line `101`?

Reply:
@amit-yb actually at line 101, i am adding double quotes around table name in case of Oracle to preserve the case for case-sensitive objects while query the source.
For example: `schema."table"`

Reply:
Ah! Sorry, I missed the double quotes.
I would still suggest to set the `FullTableName`, to the form that you can pass directly to the DB without any additional transformation.

Reply:
hmm, let me do that. I will do that while creating FullTableName filed for Oracle.